content/en/quickstart/_index.md

```diff
@@ -83,3 +81,9 @@ The error codes of the found issues are prefixed by 'IST' or 'KIA'. For the deta
 - https://istio.io/latest/docs/reference/config/analysis/ for 'IST' error codes
 - https://kiali.io/documentation/latest/validations/ for 'KIA' error codes
 ```
+
+<h3>Next Steps</h3>
+
+Read on for next steps with Tetrate Istio Distro and GetMesh.
+
+For community support, join Tetrate's <a href="https://tetr8.io/tetrate-community">#GetMesh Slack Channel</a>.
```

Comment:
The channel is called #tid-and-getmesh -- also, use the markdown syntax for links

```
[#Tid-and-GetMesh Slack Channel](add-link-here)
```

Reply:
Got it; will correct the channel name elsewhere too